### ISSUE TITLE
Upgrade code to Polymer 2.0 / Starcounter 2.4

### ISSUE THREAD

**Issue by warpech** [warpech/HelloWorld#58]
Code in this repo won't work with Starcoutner 2.4, because it still uses Polymer 1.0.

Code needs to be updated to use Polymer 2.0 and perhaps other API fixes for Starcoutner 2.4. 

When this is done, also update code snippets in the docs (https://docs.starcounter.io/v/2.4/getting-started/hello-world-tutorial/create-a-real-time-ui).

@Mackiovello would it be possible for you to do that?

**Comment by ErlendLandro** [comment]
> Code in this repo won't work with Starcoutner 2.4

Yes, that's intentional. The tutorial is only supposed to work with the current release candidate. The tutorial is for completely new users of Starcounter, so it doesn't make sense to adapt the tutorial for 2.4 since new users shouldn't use 2.4 anyway.

I know that we've discussed this before.

**Comment by warpech** [comment]
Yes, this makes sense for the repo. 

I made a separate issue for the 2.4 branch of the docs, which does not have to wait that long: https://github.com/Starcounter/Docs/issues/219
